system-addon/content-src/components/AutoMigratePrompt/AutoMigratePrompt.jsx

```diff
@@ -0,0 +1,32 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage, injectIntl} = require("react-intl");
+const {actionTypes: at, actionCreators: ac} = require("common/Actions.jsm");
+
+class AutoMigratePrompt extends React.Component {
+  render() {
+    if (!this.props.AutoMigrate.display) {
+      return null;
+    }
+
+    return this.props.AutoMigrate.stage === 0 ?
+      (<div className="migrate-prompt">
+        <span>{this.props.AutoMigrate.msg}</span>
+        <div className="confirm">
+          <button onClick={this.props.onUndoClick.bind(this)}><FormattedMessage id="migrate_undo_button" /></button>
```

Comment:
Actually, if `onUndoClick` is a prop, `this` probably shouldn't be `AutoMigratePrompt` anyway?